posthog/session_recordings/queries/session_recording_list_from_filters.py

```diff
@@ -60,7 +60,10 @@ class SessionRecordingListFromFilters:
             (duration - active_seconds) as inactive_seconds,
             sum(s.console_log_count) as console_log_count,
             sum(s.console_warn_count) as console_warn_count,
-            sum(s.console_error_count) as console_error_count
+            sum(s.console_error_count) as console_error_count,
+            -- Check if the most recent _timestamp is within five minutes of the current time
+            -- proxy for a live session
+            (max(s._timestamp) >= now() - INTERVAL 5 MINUTE) as ongoing
```

Comment:
The inactivity period (30 minutes by default) feels like a more exact interval, but 5 minutes will probably be more accurate so let's go with that for now

Reply:
David you are now officially better at making things pulse than ChatGPT 🤣